src/pudl/transform/classes.py

```diff
@@ -156,7 +163,21 @@ def multicol_transform_factory(
     col_func: ColumnTransformFunc,
     drop=True,
 ) -> MultiColumnTransformFunc:
-    """A factory for creating a multi-column transform function."""
+    """A factory function for creating multi-column transform functions.
+
+    Converts individual column transformers into multi-column transformers. This saves
```

Comment:
There are transform functions, transform parameters, and transformer classes (which gather together the functions and parameters). I've susbstituted in the actual classes as references here instead of just using words. Does that help?

Reply:
yes this is more clear to me now.